llvm/include/llvm/DebugInfo/DWARF/DWARFTypePrinter.h

```diff
@@ -9,19 +9,66 @@
 #ifndef LLVM_DEBUGINFO_DWARF_DWARFTYPEPRINTER_H
 #define LLVM_DEBUGINFO_DWARF_DWARFTYPEPRINTER_H
 
+#include "llvm/ADT/SmallString.h"
 #include "llvm/ADT/StringRef.h"
 #include "llvm/BinaryFormat/Dwarf.h"
-#include "llvm/DebugInfo/DWARF/DWARFDie.h"
+#include "llvm/DebugInfo/DWARF/DWARFUnit.h"
 
 #include <string>
 
 namespace llvm {
 
 class raw_ostream;
 
+inline std::optional<uint64_t> getLanguage(DWARFDie D) {
```

Comment:
looks like `getLanguage` and `detail::toString` aren't used at this point, so should be omitted?

Reply:
Removed.